test/functional/feature_listaccounts_pagination.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
+# Copyright (c) DeFi Blockchain Developers
+# Distributed under the MIT software license, see the accompanying
+# file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+"""Test listaccounts RPC pagination"""
+
+from test_framework.test_framework import DefiTestFramework
+from test_framework.util import assert_equal
+
+class AccountsValidatingTest(DefiTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+        self.setup_clean_chain = True
+        self.extra_args = [
+            ['-txnotokens=0', '-amkheight=50', '-eunosheight=101'],
+            ['-txnotokens=0', '-amkheight=50', '-eunosheight=101'],
+        ]
+
+    def run_test(self):
+        node = self.nodes[0]
+        node1 = self.nodes[1]
+        node.generate(101)
+        self.sync_all()
+
+        assert_equal(node.getblockcount(), 101) # eunos
+
+        # Get addresses and set up account
+        account = node.getnewaddress()
+        destination = node.getnewaddress()
+        node.utxostoaccount({account: "10@0"})
+        node.generate(1)
+        self.sync_all()
+
+        accounts = node1.listaccounts()
+
+        #make sure one account in the system
+        assert_equal(len(accounts), 1)
+
+        pagination = {
+                "start": accounts[len(accounts) - 1]["key"],
+                "including_start": True
+            }
+
+        result2 = node1.listaccounts(pagination)
+
+        #check the length of the result
+        assert_equal(len(result2), 1)
+
+        #add another account
+        account2 = node.getnewaddress()
+        destination2 = node.getnewaddress()
+        node.utxostoaccount({account2: "10@0"})
+        node.generate(1)
+        self.sync_all()
+
+        accounts = node1.listaccounts()
+        #make sure we have two account in the system
+        assert_equal(len(accounts), 2)
+
+        pagination = {
+                "start": accounts[0]["key"], #NOTE(sp) : As per the current implementation, Has to give the first element in the array, not the last.
```

Comment:
@bvbfan has the sort order changed? This behaviour is weird.

For these test conditions, I would expect it to return 2 instead of one, since `including_start: True`.

```py
assert_equal(len(result2), 1)
```